src/CSSBattle/8_ForkingCrazy/Yixuan.html

```diff
@@ -0,0 +1,65 @@
+<div class="handle"></div>
+<div class="base"></div>
+<div class="prong one"></div>
```

Comment:
for the prongs perhaps can use box shadow trick to repeat prongs

Reply:
Updated in commit f943ede, nice stuff